boards/genericSTM32G431KB.json

```diff
@@ -0,0 +1,39 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DSTM32G4xx -DSTM32G431xx",
+    "f_cpu": "170000000L",
+    "mcu": "stm32g431kbt6",
+    "product_line": "STM32G431xx",
+    "variant": "Generic_G431Kx"
```

Comment:
There is no Arduino mentioned in the framework list, so this field seems redundant.

Reply:
To be honest I have not been able to find very precise documentation on the use of the various fields in the board files. I've been inspired by existing files but I'm not sure what each field is for so if you say so this field is probably not necessary.